cluster/manifests/roles/scalyr-key-rotation-rbac.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: scalyr-key-rotation
+  namespace: visibility
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  resourceNames:
+  - scalyr-keys
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - patch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: scalyr-key-rotation
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  resourceNames:
+  - scalyr-delegation-token
+  verbs:
+  - get
+  - list
```

Comment:
`list` can't be used with `resourceNames` unfortunately, you need a separate entry for it.

Reply:
I am not even sure we need list. Let's remove it and see if we can work without it.